nupic/research/frameworks/vernon/mixins/prune_low_snr.py

```diff
@@ -100,7 +100,10 @@ def pre_epoch(self):
 
     def run_epoch(self):
         results = super().run_epoch()
-        vdrop_data = self.model.module.vdrop_data
+        model = self.model
+        if hasattr(model, "module"):
+            model = model.module
```

Comment:
Are you sure these changes are compatible with how this mixin is used elsewhere? It seems like you've made it custom to fit your case: the line you removed (old 103) would be equivalent of `self.model.module.module.vdrop_data` under your new implementation, which looks like an inconsistency to me.

Reply:
Good catch, fixed